### ISSUE TITLE
Alcatraz and Plugins will stop working in XCode 8

### ISSUE THREAD

**Issue by gargantuan** [MarcoSero/EditorConfig-Xcode#6]
From what I can tell by reading this [Alcatraz issue](https://github.com/alcatraz/Alcatraz/issues/475), Apple are pushing Extensions over Plugins. The reasoning seems sensible, but at the moment it's looking like plugins will stop working soon.

Are there any plans to port this plugin to an extension?

**Comment by MarcoSero** [comment]
No, at the moment I have no plans/time to port it over I'm afraid.

**Comment by willin** [comment]
are there any other solutions?
